src/locales/en.json

```diff
@@ -127,5 +127,14 @@
   },
   "errorDialog": {
     "title": "Error"
+  },
+  "shortcutsDialog": {
+    "title": "Keyboard shortcuts",
+    "shapes": "Shapes",
+    "editor": "Editor",
+    "view": "View",
+    "blog": "Read the Blog",
+    "howto": "Check out the How To",
+    "github": "Found an issue? Submit"
```

Comment:
I'm out of ideas on what to write here.. what should be the links? Anyone?!

Reply:
blog & github is cool. For the `howto`, maybe:

- "Check out the help guides"
- "Check out our help"

Reply:
- `Read our blog`
- `Follow the guides`
- `See the source code`

Reply:
@kbariotis the github thing is for bug/issues, not really about source code.

Reply:
Check them out.. I had a combination of the suggestions.